Doc/library/stdtypes.rst

```diff
@@ -4618,6 +4605,45 @@ The constructors for both classes work the same:
    :meth:`discard` methods may be a set.  To support searching for an equivalent
    frozenset, a temporary one is created from *elem*.
 
+.. class:: frozenset([iterable])
+
+   Return a new frozenset object whose elements are taken from *iterable*.
+
+   Instances of :class:`frozenset` provide the following :class:`set` operations:
+
+   .. method:: len(s)
```

Comment:
This would render quite ugly IMO. If users use a link to `frozenset.isdisjoint` in their Sphinx project, then it will go there and then the user wouldn't have any proper description.